src/backend/dx12/src/lib.rs

```diff
@@ -1047,6 +1047,23 @@ impl hal::Instance<Backend> for Instance {
             //TODO: find a way to get a tighter bound?
             let sample_count_mask = 0x3F;
 
+            // Theoretically vram limited, but in practice 2^20 is the limit
+            let tier3_practical_descriptor_limit = 1 << 20;
```

Comment:
This particular bit was derived from "word of mouth" issues, not aware of a hard cite.